### PR TITLE
Add build option for Ublox

### PR DESCRIPTION
**Pull request by Hwurzburg**
mate to #20528

### CODE REVIEW

**Review by amilcarlucas — APPROVED** [review]

File: Tools/scripts/build_options.py [hunk 1]
```diff
@@ -178,6 +178,7 @@ def __init__(self,
     Feature('Other', 'NMEA_OUTPUT', 'HAL_NMEA_OUTPUT_ENABLED', 'Enable NMEA Output', 0, None),
     Feature('Other', 'BARO_WIND_COMP', 'HAL_BARO_WIND_COMP_ENABLED', 'Enable Baro Wind Compensation', 0, None),
 
+    Feature('GPS Drivers', 'Ublox', 'AP_GPS_UBLOX_ENABLED', 'Enable Ublox GPS', 1, 'None'),
```
Comment (peterbarker) [inline Tools/scripts/build_options.py:181]:
```suggestion
    Feature('GPS Drivers', 'UBLOX', 'AP_GPS_UBLOX_ENABLED', 'Enable u-blox GPS', 1, None),
```

(that's apparently they way they branch their GPS, caps-wise...)

**Review by peterbarker — CHANGES_REQUESTED** [review]
Thanks for doing this, Henry.

I wonder if we really should default this to on.... probably....

**Review by peterbarker — CHANGES_REQUESTED** [review]
commit message needs to be prefixed with `Tools: build_options.py: `

**Comment by Hwurzburg** [comment]
duh, sorry...fixed

**Review by peterbarker — APPROVED** [review]
LGTM

**Pull request merged**